build.gradle

```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.library'
+
+
+def versionMajor = 1
```

Comment:
let's set to 0.1.0 first, after we did a round of testing with some phones and are confident, we can make it 1.0.

Reply:
ok